jaxrs-tck/src/main/java/jakarta/ws/rs/tck/common/JAXRSCommonClient.java

```diff
@@ -533,15 +533,19 @@ protected static String buildHeaderMediaType(String header, MediaType type) {
     return sb.toString();
   }
 
-  public static String editWebXmlString(InputStream inStream) throws IOException{
+  public static String toString(InputStream inStream) throws IOException{
     String line;
-    String webXmlTemplate = "";
+    String str = "";
     try (BufferedReader bufReader = new BufferedReader(new InputStreamReader(inStream, StandardCharsets.UTF_8))) {
       while ((line = bufReader.readLine()) != null) {
-        webXmlTemplate = webXmlTemplate + line + System.lineSeparator();
+        str = str + line + System.lineSeparator();
```

Comment:
I know this is just a TCK, but should we replace this by a `StringBuilder`?

Reply:
Sure - I think I just left the existing logic alone there, but we could optimize, modernize, and economize it to be:
```
  public static String toString(InputStream inStream) throws IOException{
    try (BufferedReader bufReader = new BufferedReader(new InputStreamReader(inStream, StandardCharsets.UTF_8))) {
      return bufReader.lines().collect(Collectors.joining(System.lineSeparator()));
    }
  }

  public static String editWebXmlString(InputStream inStream) throws IOException{
    return toString(inStream).replaceAll("servlet_adaptor", servletAdaptor);
  }
```